examples/sequence_kernel_example.py

```diff
@@ -0,0 +1,69 @@
+import torch
+import eigenpro.kernels as kernels
+import eigenpro.models.sharded_kernel_machine as skm
+import eigenpro.solver as solver
+import eigenpro.utils.device as dev
+from eigenpro.data.utils import protein_2_1hot
+from datasets import load_dataset
```

Comment:
what is the `datasets` package? is this huggingface?

Reply:
yes it is hugging face. quite handy